### PR TITLE
Campbell colors

### PR DESCRIPTION
**Pull request by fdncred**
Right now there is a hard coded theme named `default` and a theme named `campbell` which is microsoft's windows terminal default theme. So, i added `--campbell` as a way to change the theme from `default` to `campbell`. I see `--campbell`  as a temporary parameter for testing  and vetting out what we should really do.

### CODE REVIEW

**Comment by fdncred** [comment]
What is the deal with `ci/circleci` taking forEVER?

File: crates/nu-cli/tests/format_conversions/html.rs [hunk 1]
```diff
@@ -20,12 +20,12 @@ fn out_html_table() {
     let actual = nu!(
         cwd: ".", pipeline(
         r#"
-            echo '{"name": "jason"}' | from json | to html
+            echo '{"name": "darren"}' | from json | to html
```
Comment (jonathandturner) [inline crates/nu-cli/tests/format_conversions/html.rs:23]: lol

File: crates/nu-cli/src/commands/to_html.rs [hunk 2]
```diff
@@ -581,7 +632,8 @@ mod tests {
         let where_help = r"<html><style>body { background-color:black;color:white; }</style><body>Filter table to match the condition.<br><br>Usage:<br>  &gt; where &lt;condition&gt; {flags} <br><br>Parameters:<br>  &lt;condition&gt; the condition that must match<br><br>Flags:<br>  -h, --help: Display this help message<br><br>Examples:<br>  List all files in the current directory with sizes greater than 2kb<br>  &gt; [1;36mls[0m[37m | [0m[1;36mwhere[0m[37m [0m[1;33msize[0m[37m [0m[33m&gt;[0m[37m [0m[1;35m2[0m[1;36mkb[0m<br><br>  List only the files in the current directory<br>  &gt; [1;36mls[0m[37m | [0m[1;36mwhere[0m[37m [0m[1;33mtype[0m[37m [0m[33m==[0m[37m [0m[32mFile[0m<br><br>  List all files with names that contain &quot;Car&quot;<br>  &gt; [1;36mls[0m[37m | [0m[1;36mwhere[0m[37m [0m[1;33mname[0m[37m [0m[33m=~[0m[37m [0m[32m&quot;Car&quot;[0m<br><br>  List all files that were modified in the last two months<br>  &gt; [1;36mls[0m[37m | [0m[1;36mwhere[0m[37m [0m[1;33mmodified[0m[37m [0m[33m&lt;=[0m[37m [0m[1;35m2[0m[1;36mM[0m<br><br></body></html>".to_string();
```
Comment (jonathandturner) [inline crates/nu-cli/src/commands/to_html.rs:632]: I missed these before... I wonder if for the larger tests we should move them out to a separate file?

File: crates/nu-cli/tests/format_conversions/html.rs (hunk 1 above)
Comment (fdncred) [reply crates/nu-cli/tests/format_conversions/html.rs:23]: This was me trying to make a small change in order to get CI to work again. :)

File: crates/nu-cli/src/commands/to_html.rs (hunk 2 above)
Comment (fdncred) [reply crates/nu-cli/src/commands/to_html.rs:632]: Sure, I'll work on this.

**Pull request merged**